README.md

```diff
@@ -1,6 +1,10 @@
 ![UI5 logo](./docs/images/UI5_logo_wide.png)
 
 # UI5 Tooling
```

Comment:
Why is this text different from e.g. https://github.com/SAP/ui5-fs/pull/506/files ?

Reply:
Thanks!